src/sonic-yang-models/yang-models/sonic-sflow.yang

```diff
@@ -32,10 +32,28 @@ module sonic-sflow{
 
     description "SFLOW yang Module for SONiC OS";
 
+    revision 2023-04-11 {
```

Comment:
In https://github.com/openconfig/public/blob/master/release/models/sampling/openconfig-sampling-sflow.yang egress-sampling-rate supported at global level and per interface level ? How do we map sonic-sflow.yang to https://github.com/openconfig/public/blob/master/release/models/sampling/openconfig-sampling-sflow.yang ? or we don't support egress-sampling-rate configuration at all in SONIC?